### ISSUE TITLE
--no-debug doesn't mitigate the logging overhead

### ISSUE THREAD

**Issue by vpodzime** [storaged-project/storaged#129]
This chunk of code speaks for itself:
```
  /* TODO: this hammer is too big - it would be a lot better to configure the
   *       logging routines and avoid printf(3) overhead and so on
   */
  if (opt_no_debug)
    {
      gint dev_null_fd;
      dev_null_fd = open ("/dev/null", O_RDWR);
      if (dev_null_fd >= 0)
        {
          dup2 (dev_null_fd, STDIN_FILENO);
          dup2 (dev_null_fd, STDOUT_FILENO);
          dup2 (dev_null_fd, STDERR_FILENO);
          close (dev_null_fd);
        }
```

**Comment by vpodzime** [comment]
Addressed by the PR #143

**Comment by vpodzime** [comment]
> Addressed by the PR #143

Merged.
